Dockerfile

```diff
@@ -15,6 +15,8 @@ RUN apt-get install -y --no-install-recommends \
         apt-get clean && \
         rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
 
+RUN apt-get update && apt-get install iputils-ping 
```

Comment:
This should be added to the larger `apt-get install` block to improve build time

Reply:
iputils-ping needs to be done after the apt-get update due to a weird issue how it sets itself up within docker.  You can find a bunch of examples of threads discussing this issue - e.g. https://superuser.com/questions/718916/problems-installing-ping-in-docker

Reply:
Those examples seem to only state that you need to have an `apt-get update` in front. Which makes sense because the base image doesn't have that cache built.

However, the main install line that is already in the image contains that. Does it fail to build? Or fail to run? Can you provide me with a plugin that would allow me to test?

Reply:
Those examples seem to only state that you need to have an apt-get update in front. Which makes sense because the base image doesn't have that cache built.

However, the main install line that is already in the image contains that. Does it fail to build? Or fail to run? Can you provide me with a plugin that would allow me to test?

Reply:
it fails to run.  you can test it by attaching to the docker container and running attempting to run the ping command.

The plugin that i was using that used this was the homebridge-people plugin (although i stopped using it since this for other reasons, and also i think IOS11 HomeKit supports person detection natively).  Its also used by the lgtv plugin to turn on the TV via a network IP ping.

Reply:
Try using the branch in #6. I just tried and it works for me...